bcda/logging/middleware.go

```diff
@@ -77,23 +77,15 @@ type ResourceTypeLogger struct {
 
 func (rl *ResourceTypeLogger) LogJobResourceType(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		next.ServeHTTP(w, r)
-
 		jobKey, err := rl.extractJobKey(r)
 		if err != nil {
-			log.API.Error(err)
-			return
-		}
-		// Note: could split this out into a function for adding to the context log
-		entry, ok := middleware.GetLogEntry(r).(*log.StructuredLoggerEntry)
-		if !ok {
-			log.API.Error("Incorrect type of logger used in request context")
+			logger := log.GetCtxLogger(r.Context())
```

Comment:
Why are some logs made using the logger while others aren't?

Reply:
We should use the context logger in most cases if we have access to a context or http.Request object. I tried to do that in this Pr except for the `writer` files which will require a more involved refactoring to get access to the request context. I'll create a ticket for that.

Reply:
Followup ticket here: https://jira.cms.gov/browse/BCDA-7581

cc. @laurenkrugen-navapbc